src/kernel/elf.zig

```diff
@@ -639,7 +646,7 @@ test "getName" {
     // The entire ELF test data. The header, program header, two section headers and the section name (with the null terminator)
     var section_name = "some_section";
     var string_section_name = "strings";
-    const data = testInitData(section_name, string_section_name, .Executable, 0, undefined, undefined, undefined, undefined, undefined);
+    const data = try testInitData(testing.allocator, section_name, string_section_name, .Executable, 0, undefined, undefined, undefined, undefined, undefined);
```

Comment:
Not sure it is a good idea to use `undefined` as parameters, might be better to use `0xAA` directly so it is more defined.

Reply:
The point of `undefined` is that we don't care about the value. Using `0xAA` would imply that we care about the value.